va/drm/va_drm_utils.c

```diff
@@ -69,7 +69,7 @@ VA_DRM_GetNumCandidates(VADriverContextP ctx, int * num_candidates)
     }
     drmFreeVersion(drm_version);
     *num_candidates = num_of_candidate;
-    return VA_STATUS_SUCCESS;
+    return num_of_candidate ? VA_STATUS_SUCCESS : VA_STATUS_ERROR_UNKNOWN;
```

Comment:
*num_of_candidate?

Reply:
no:). num_of_candidate is local variable declared int. num_candidates is function argument declared int*. I would suggest to use simpler and more different names to avoid misreading:).

Reply:
sorry，a mistake

Reply:
So should I roll back to original version?

Reply:
OK, now I changed num_of_candidate to count in my cl.
Please take another look, thanks!